### PR TITLE
Simple combo samples

### LINKED ISSUE #2828: Combo single selection sample not behaving as expected
### Description  
[Single selection sample](https://www.infragistics.com/products/ignite-ui-angular/angular/components/combo#single-selection) does not work as intended. When having one item selected, the sample does not prevent another one to be selected at the same time.
The reason for that is the use of an outdated event `selectionChange`.

### LINKED ISSUE #2834: Add samples for new igxSimpleCombo component
### Description  
Add samples for new `igxSimpleCombo` component

### PR DESCRIPTION
**Pull request by wnvko**
Add simple combo samples. Also removes combo's single selection and cascading samples.
Closes #2834 
Closes #2828

### CODE REVIEW

**Comment by wnvko** [comment]
All new added samples need designer touch. @Lipata please assign someone from design team once samples are dev done.

**Review by Lipata — APPROVED** [review]

**Review by Lipata — APPROVED** [review]

File: src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import * as core from '@angular/core';
```
Comment (jackofdiamond5) [inline src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts:1]: Is there a reason why we're importing everything like this?

File: src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    styleUrls: ['simple-combo-cascading.component.scss']
+})
+export class SimpleComboCascadingComponent implements core.OnInit {
+    public selectedCountry: Country;
+    public selectedRegion: Region;
+    public selectedCity: City;
+    public countriesData: Country[];
+    public regionData: Region[] = [];
+    public citiesData: City[] = [];
+    ngOnInit(): void {
```
Comment (jackofdiamond5) [inline src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts:17]:
```suggestion
    public ngOnInit(): void {
```

File: src/app/lists/combo/simple-combo-main/simple-combo-main.component.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import * as core from '@angular/core';
```
Comment (jackofdiamond5) [inline src/app/lists/combo/simple-combo-main/simple-combo-main.component.ts:1]: Same here?

File: src/app/lists/combo/simple-combo-styling/simple-combo-styling.component.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import * as core from '@angular/core';
```
Comment (jackofdiamond5) [inline src/app/lists/combo/simple-combo-styling/simple-combo-styling.component.ts:1]: And here.

File: src/app/lists/combo/simple-combo-usage/simple-combo-usage.component.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import * as core from '@angular/core';
```
Comment (jackofdiamond5) [inline src/app/lists/combo/simple-combo-usage/simple-combo-usage.component.ts:1]: The way I see it in every component file of the simple combo we're importing * from angular/core.

File: src/app/lists/combo/simple-combo-usage/simple-combo-usage.component.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+@core.Component({
+    selector: 'app-simple-combo-usage',
+    templateUrl: 'simple-combo-usage.component.html',
+    styleUrls: ['simple-combo-usage.component.scss']
+})
+export class SimpleComboUsageComponent implements core.OnInit {
+    public cities: City[];
+    public selectedCity: number;
+    ngOnInit(): void {
```
Comment (jackofdiamond5) [inline src/app/lists/combo/simple-combo-usage/simple-combo-usage.component.ts:12]:
```suggestion
    public ngOnInit(): void {
```

File: src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts (hunk 1 above)
Comment (wnvko) [reply src/app/lists/combo/simple-combo-cascading/simple-combo-cascading.component.ts]: have no idea where all these came from
resolved now

**Pull request merged**